### DISCUSSION TITLE
The added organ in MoBi can not be found in PK-sim.

### DISCUSSION THREAD

**Discussion by JiaojiaoChen123** [Open-Systems-Pharmacology/Forum#1458] [Questions & Problems]
Dear professor, 

I want to add a virtual organ in MoBi according to the manual instructions and the discussion (https://github.com/Open-Systems-Pharmacology/Forum/discussions/1180). After adding the organ, there was no error in the simulation, but the blood concentration has not alotered. I tried to import the MoBi file into PK-sim and found that there were no new organs. And the tumor model (https://github.com/Open-Systems-Pharmacology/Forum/discussions/1180) can find the tumor organs they developed. What is the reason for this condition?

My model: ![image](https://user-images.githubusercontent.com/95741315/220607075-f98bf12e-7b77-4e3c-a97b-ca3f664198a6.png)
Tumor model: ![image](https://user-images.githubusercontent.com/95741315/220607311-4b04988d-7fe8-477d-875b-ad016c28610b.png)

Thank you in advance,
Jiaojiao

**Comment by PavelBal** [comment]
Make sure that your molecule is `present` in the new organ. For this, you have to extend the Molecules Start Values BB:

![image](https://user-images.githubusercontent.com/25903040/220609616-2530bac9-82aa-4053-af70-38a50db8f28b.png)

**Reply by JiaojiaoChen123** [reply]
Thanks for your reply,and the problem was solved.
